### PR TITLE
chore: Create pr-assign-team-label.yml

### PR DESCRIPTION
**Pull request by PeerRich**
testing, goal: tag each PR according to our organisation

### CODE REVIEW

**Review by pumfleet — DISMISSED** [review]
Nice

**Review by pumfleet — DISMISSED** [review]
It’s not working, it says the token is missing

File: .github/workflows/pr-assign-team-label.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      - main
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
+        with:
+          repo-token: ${{ secrets.TOKEN }}
```
Comment (PeerRich) [inline .github/workflows/pr-assign-team-label.yml:14]: where do i find this?

Comment (leog) [reply .github/workflows/pr-assign-team-label.yml:14]:
```suggestion
          repo-token: ${{ secrets. GITHUB_TOKEN }}
```

File: .github/workflows/pr-assign-team-label.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+      - main
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
+        with:
+          repo-token: ${{ secrets. GITHUB_TOKEN }}
```
Comment (PeerRich) [inline .github/workflows/pr-assign-team-label.yml:14]:
```suggestion
          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

File: .github/workflows/pr-assign-team-label.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+      - main
+
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```
Comment (PeerRich) [inline .github/workflows/pr-assign-team-label.yml:14]:
```suggestion
          repo-token: ${{ secrets.GH_ACCESS_TOKEN }}
```

File: .github/workflows/pr-assign-team-label.yml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: equitybee/team-label-action@main
+        with:
+          repo-token: ${{ secrets.GH_ACCESS_TOKEN }}
+          organization-name: calcom
+          ignore-labels: "insights, i18n, organizations, workflows, embed, webhooks, event types, routing forms, billing, ci, docs, ui, booking page, api, automated tests, seats, bookings, caldav, teams, performance, emails, app store, authentication"
```
Comment (PeerRich) [inline .github/workflows/pr-assign-team-label.yml:16]: hmmm ignoring wont work... i wonder if we can instead fork this action and limit it only to "cal-com" label?

File: .github/workflows/pr-assign-team-label.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  team-labels:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: calcom/team-label-action@main
+        with:
+          repo-token: ${{ secrets.GH_ACCESS_TOKEN }}
+          organization-name: calcom
+          only-labels: "cal-com"
```
Comment (PeerRich) [inline .github/workflows/pr-assign-team-label.yml:16]: only-label not quite working yet: https://github.com/calcom/team-label-action/blob/0d3f80de66445d54370716df2b8e834eb40f75b7/src/main.ts#L30

work in progress

**Review by leog — APPROVED** [review]
LGTM

**Pull request merged**